scripts/ci/clang_ignore.txt

```diff
@@ -0,0 +1,9 @@
+closure/goog/locale/defaultlocalenameconstants.js
```

Comment:
don't these need to turn off clang format internally too?  Shouldn't this be in the source?